nautobot/docs/configuration/authentication/sso.md

```diff
@@ -361,3 +361,54 @@ This should be the URL that is mapped to the "Log in" button on the top right of
 ---
 
 Be sure to configure [`EXTERNAL_AUTH_DEFAULT_GROUPS`](../../configuration/optional-settings.md#external_auth_default_groups) and [`EXTERNAL_AUTH_DEFAULT_PERMISSIONS`](../../configuration/optional-settings.md#external_auth_default_permissions) next.
```

Comment:
Does this note also apply to the Azure AD case? It was the end of the file previously, and the `---` above it suggests that it's meant as a general note, not specific to the preceding section. Might make sense to keep it at the end of the file.

Reply:
This section is for the SAML section above.

Reply:
I've added some dashes for the bottom portion to hopefully make it clearer that it's the final steps for setting up Azure to work with Nautobot.